main/RFXNames.cpp

```diff
@@ -2759,6 +2760,11 @@ bool GetLightCommand(
 			cmd=Limitless_SetRGBColour;
 			return true;
 		}
+		else if (switchcmd == "Set HexRGBColor")
+		{
+			cmd = Limitless_SetHEXColour;
```

Comment:
Sorry my mistake, "Limitless_SetHEXColour" was a remain from a test on another branch.
I have added it to push hex RGB value (from API/Dashboard request)  to the hardware without the HSV conversion.
Arilux AL-C0x doesn't have brightness control, so current method SetHUE+SetBrightness (in webserverp.cpp)  doesn't work well with this hardware.

I've created Limitless_SetHEXColour to doesn't break compatibility with other "Limiteless" based devices that use Limitless_SetRGBColour with a hue color value.

I will delete it in my next commit. :)


My HSV->RGB conversion is based on MySensorBase.cpp/BleBox.cpp/ZWaveBase.cpp
Do you want me to change something ?